arangod/RestHandler/RestVocbaseBaseHandler.cpp

```diff
@@ -325,7 +325,7 @@ void RestVocbaseBaseHandler::generate20x(arangodb::OperationResult const& result
           assembleDocumentId(collectionName,
                              slice.get(StaticStrings::KeyString).copyString(), true));
       _response->setHeaderNC(StaticStrings::Location,
-                             std::string("/_db/" + _request->databaseName() +
+                             std::string("/_db/" + StringUtils::encodeURIComponent(_request->databaseName()) +
```

Comment:
Sometimes we use `encodeURIComponent`, sometimes `urlEncode`. What's the difference? Can we harmonize this?

Reply:
Now unified it.